docs/reference/badge_spec.md

```diff
@@ -70,21 +70,21 @@ This badge would be added on a case by case basis, and would likely only be adde
 
 As with all other badges this badge may be removed at any time, and if you, as a package maintainer have updated a package and did not see this badge removed automatically, or within a reasonable time, feel free to [create an issue](https://github.com/pulsar-edit/package-backend/issues) asking for it to be removed.
 
-## Made for Pulsar!
+### Made for Pulsar!
 
-The `Made for Pulsar!` badge is automatically applied to any community packages that have been published to Pulsar through the Pulsar Package Registry. Unlike most other badges, which are applied to the package itself and thus saved in the Pulsar Package Registry Database, the `Made for Pulsar!` badge is applied dynamically at the time it is requested from the Pulsar Package Registry. Since this badge is applied dynamically there is no real way, currently, to opt out of it's usage if a community package maintainer wished to do so.
+The `Made for Pulsar!` badge is automatically applied to any community packages that have been published to Pulsar through the PPR. Unlike most other badges, which are applied to the package itself and thus saved in the PPR Database, the `Made for Pulsar!` badge is applied dynamically at the time it is requested from the PPR. Since this badge is applied dynamically there is no real way, currently, to opt out of it's usage if a community package maintainer wished to do so.
```

Comment:
```suggestion
The `Made for Pulsar!` badge is automatically applied to any community packages that have been published to Pulsar through the PPR. Unlike most other badges, which are applied to the package itself and thus saved in the PPR Database, the `Made for Pulsar!` badge is applied dynamically at the time it is requested from the PPR. Since this badge is applied dynamically there is no real way, currently, to opt out of its usage if a community package maintainer wished to do so.
```